docs/src/testing/test-collection.md

```diff
@@ -1,95 +1,111 @@
-## Test Collection
+# How Tests Are Collected
 
-`snforge` considers all functions in your project marked with `#[test]` attribute as tests.
-By default, test functions run without any arguments.
-However, adding any arguments to function signature will enable [fuzz testing](../snforge-advanced-features/fuzz-testing.md) for this
-test case.
```

Comment:
Do we want to get rid of this information, is it written somewhere else?

Reply:
We have a separate page on fuzz testing where we explain how to enable them. I don't think the page on test collection benefits from having this explained.